deps.bzl

```diff
@@ -543,6 +543,12 @@ def go_dependencies():
         sum = "h1:vV6w1AhK4VMnhBno/TPVCoK9U/LP0PkLCS9tbxHdi/U=",
         version = "v1.1.1",
     )
+    go_repository(
+        name = "com_github_dimuska139_go_email_normalizer",
+        importpath = "github.com/dimuska139/go-email-normalizer",
+        sum = "h1:pJNZnU7uS9MRoYqpoir05B+bCYXrS9sPGE4G1o9EDA8=",
+        version = "v1.2.1",
+    )
```

Comment:
I know the open source ecosystem runs on herculean feats of individuals, but I'm still not a fan of random people's not-particularly-widely-used libraries being used in PII sensitive contexts (or really just on the backend in general). If you've audited this and trust it, LGTM, but otherwise, I'd fork it or just make the our own minimal version with the normalizations we care about.

I noted that email normalizing is a fraught endeavor, and I'd argue that farming it out to a random dependency is also a fraught solution.

Reply:
I share your general skepticism, which is why I also audited it. RE: should we create our own fork, I was thinking that Bazel deps provides a sufficient mechanism, since the version + checksum validate that this is the only version that we'll ever use unless we explicitly upgrade it. Is there a way we could more firmly lock in at this version?

Reply:
I think code review is the best/easiest tool we have available, the dependency should only change versions if we `go get -u ...` some other dep that requires a newer version, which seems unlikely.

Reply:
SGTM.